sdk/core/azure-core/inc/azure/core/cryptography/hash.hpp

```diff
@@ -97,12 +97,12 @@ namespace Azure { namespace Core { namespace Cryptography {
     std::vector<uint8_t> Final() { return Final(nullptr, 0); };
 
     /**
-     * @brief Cleanup any state when destroying the instance of @Hash.
+     * @brief Cleanup any state when destroying the instance of #Azure::Core::Cryptography::Hash.
      */
     virtual ~Hash(){};
```

Comment:
```suggestion
    virtual ~Hash() = default;
```

Reply:
I would like to understand these subtle differences better. Why do we want to do this over the existing?

Reply:
(1) The `;` in `virtual ~Hash(){};` is extraneous noise.
(2) `= default` is "better" than `{}` in general since it sometimes (though not in this case) produces trivial special members which have performance advantages over non-trivial members. (`= default` is also a nice "this space intentionally left blank" making it explicit that someone didn't forget to fill in the `{}`.) I prefer to teach using `= default` even in cases like this one where there's no technical reason to get folks' fingers in the habit so they do the right thing when it does matter.